src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/TablePartitioningStrategy.java

```diff
@@ -52,88 +44,14 @@
      */
     void validateCompatiblePrimaryKeys(PrimaryKey virtualPrimaryKey, PrimaryKey physicalPrimaryKey);
 
+    /**
+     * Parses a given physical hash key value into the record's MT context and virtual table name.
+     */
+    MtContextAndTable toContextAndTable(ScalarAttributeType physicalHashKeyType, AttributeValue physicalHashKeyValue);
+
```

Comment:
To scan all tenants, we need a way to parse the MT context and virtual table name from a given physical record